recipes/libfaketime/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "libfaketime" %}
+{% set version = "0.9.8" %}
+{% set sha256 = "06288237cd5890eca148489e5b904ed852ed0ffa8424bfb479342f4daa8442a3" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wolfcw/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "make install PREFIX=$PREFIX"
```

Comment:
Is there a `make check` that can be run?

Reply:
No, there is a `make test` though. I added that to the `build.sh` now.